src/planscape/attributes/load.py

```diff
@@ -0,0 +1,132 @@
+import json
+import numpy as np
+import os
+import rasterio
+import subprocess
+
+from attributes.models import Attribute, BaseAttribute
+from base.conditions import convert_nodata_to_nan
+from eval.raster_data import RasterData
+from planscape import settings
+
+
+# Loads the raster in the raster path to the DB table,
+# attributes_attributeraster.
+# Of note: Multiple rasters will be written to the DB, each with dimensions,
+# 256 x 256.
+# The names of the rasters in the DB will be the same as the name of the
+# rasters on disk (including the format indicator at the end).
+def _load_raster(raster_path):
+    cmds = 'export PGPASSWORD=' + settings.PLANSCAPE_DATABASE_PASSWORD + '; raster2pgsql -s 9822 -a -I -C -Y -f raster -n name -t 256x256 ' + \
+        raster_path + ' public.attributes_attributeraster | psql -U planscape -d planscape -h ' + settings.PLANSCAPE_DATABASE_HOST + ' -p 5432'
```

Comment:
5432 is the default port for psql; you probably don't need to specify this.  Instead (optionally) if you think that the postgres could be responding on a different port (e.g. per some env variable), you could reference that env variable.

Reply:
Note that the same applies in the comment in models.py, if you choose to not keep -p 5432

Reply:
Fixed. I've also replaced 9822 with CRS_FOR_RASTERS.